src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/EnumBuilder.cs

```diff
@@ -184,13 +184,25 @@ public override Type[] GetNestedTypes(BindingFlags bindingAttr)
             return m_typeBuilder.GetNestedType(name, bindingAttr);
         }
 
-        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
+        [DynamicallyAccessedMembers(
+            DynamicallyAccessedMemberTypes.PublicFields | DynamicallyAccessedMemberTypes.NonPublicFields |
```

Comment:
Can we make an internal const value for this that gets shared everywhere? That way we don't need 6 dense lines in all these methods.